src/metatrain/experimental/gap/model.py

```diff
@@ -37,9 +37,14 @@ def __init__(self, model_hypers: Dict, dataset_info: DatasetInfo) -> None:
 
         # Check capabilities
         for target in dataset_info.targets.values():
-            if target.quantity != "energy":
+            if not (
+                target.is_scalar
+                and target.quantity == "energy"
+                and "atom" not in target.layout.block(0).samples.names
```

Comment:
here a stupid question becouse i do not know the code.  when `atom` is in `target.layout.block(0).samples.names`, 
  `target.quantity` can still be `energy`? if this is the case i can see a case where the error becomes:
  
  ```"GAP only supports total-energy-like outputs, but a energy was provided"```
  
  which do not make much sense. 
  If this is the case split this error message for the case `"atom" not in target.layout.block(0).samples.names`.